eve/main.yml

```diff
@@ -30,7 +30,23 @@ stages:
         retryFetch: true
         haltOnFailure: true
     - ShellCommand:
-        name: 'Run test-suite'
+        name: 'Init node'
         haltOnFailure: true
         command: |-
-          bash ./vendor/bash_unit/bash_unit tests/single-node/test.sh
+          bash tests/single-node/init.sh
+    - ShellCommand:
+        name: 'Run the install test suite'
+        haltOnFailure: true
+        usePTY: true
+        command: |-
+          tox -e test-install -- -s
```

Comment:
I guess this will be too little.

Reply:
I don't understand this one. Could you explain a little bit ?

Reply:
Well, this now runs the `test-install` scenario, but as described above we'll have multiple installation scenarios. It's still unclear to me how this ought to be handled.

Reply:
Never mind.

Reply:
What I don't like is how there are many tox targets, one for `test-install`, one for `test`,... I'd rather have one `tox` target, and use `{posargs}` to select relevant tests, then adjust the invocations of `tox` accordingly.